core/web/presenters/job.go

```diff
@@ -330,6 +330,7 @@ type BlockhashStoreSpec struct {
 	CoordinatorV2PlusAddress       *ethkey.EIP55Address  `json:"coordinatorV2PlusAddress"`
 	WaitBlocks                     int32                 `json:"waitBlocks"`
 	LookbackBlocks                 int32                 `json:"lookbackBlocks"`
+	HeartbeatPeriodTime            int32                 `json:"heartbeatPeriodTime"`
```

Comment:
I think we should be consistent with other duration fields in the repo and use `time.Duration` instead of `int32` for this.

```suggestion
	HeartbeatPeriod            time.Duration                 `json:"heartbeatPeriodTime"`
```

Reply:
See `PollPeriod` or `RunTimeout` in this job spec as examples.

Reply:
If we do that, we may not be able to set the HeartbeatPeriodTime as 0 to disable it. This is the GraphQL schema:
```
heartbeatPeriodTime: Int!
pollPeriod: String!
runTimeout: String!
```

Reply:
Refactored heartbeatPeriodTime (int32 seconds) to heartbeatPeriod time.Duration